src/seahawk_rov/seahawk_rov/helper_functions.py

```diff
@@ -56,3 +56,10 @@ def clamp(num, minimum, maximum):
   minimised = min(maximised, maximum)
 
   return minimised
+
+# deadzone helper function
+# if within range set value to something; else do nothing
+def deadzone(num, min, max, value=0):
```

Comment:
Names `num` and `value` aren't the best. It's not clear which one is being deadzoned.

The code looks like it will select the `value` when the `num` is out of bounds. It should default to 0 with no option to change. This function should operate in normalized space, not the PWM frequency range. There's another converter for that step.